operation.go

```diff
@@ -330,7 +329,7 @@ func (o *Operation) Password(prompt string) ([]byte, error) {
 	o.t.EnterRawMode()
 	defer o.t.ExitRawMode()
 
-	b, err := terminal.ReadPassword(int(os.Stdin.Fd()))
+	b, err := terminal.ReadPassword(int(o.cfg.Stdin.Fd()))
```

Comment:
@lunixbochs I haven't dig out a lot about how it implement, I'm not sure it's as simple as stop echoing characters.
Anyway, it seems not breaking anything. Thanks for you pr!